src/main/java/com/github/redouane59/twitter/TwitterClient.java

```diff
@@ -251,12 +253,20 @@ public Tweet postTweet(String text) {
 
   @Override
   public Tweet postTweet(String text, String inReplyToStatusId) {
+    return this.postTweet(text, inReplyToStatusId, null);
+  }
+
+  @Override
+  public Tweet postTweet(String text, String inReplyToStatusId, String mediaId) {
```

Comment:
Would be more clear if field `mediaId` will be renamed to `mediaIds`. And ideally it would be if multiple mediaIds can be received. for example through varargs, like this:
 ```java
public Tweet postTweet(String text, String inReplyToStatusId, String... mediaIds) {
```

Reply:
Yes that's a good idea, I'm just a lot lazy to implement it now knowing that the V2 endpoint will come soon and that I will have to re-develop everything. But don't hesitate to submit a pull request !